filters/imagefilter.go

```diff
@@ -101,3 +103,10 @@ func transformImage(out *io.PipeWriter, image *bimg.Image, opts *bimg.Options) e
 
 	return err
 }
+
+func applyDefaults(o *bimg.Options) *bimg.Options {
+	if o.Quality == 0 {
```

Comment:
I think that the default quality for cropping for example should be 100. Nobody expects to loose information—except the cropped parts—while cropping.